client/src/ui/Gauge.tsx

```diff
@@ -24,7 +26,22 @@ export default function Gauge({ rating, label }: GaugeProps) {
   useEffect(() => {
     if (rating && !isNaN(rating)) {
       let percentage = 20 * rating; // rating is 1-5
-      let color = `hsl(212, 100%, ${86 - percentage * 0.36}%)`;
+      let color;
+
+      let red = `hsl(4, 100%, 71%)`;
+      let yellow = `hsl(47, 94%, 58%)`;
+      let green = `hsl(101, 64%, 43%)`;
+
+      let ratingRounded = parseFloat(rating.toFixed(1));
+
+      if (0 <= ratingRounded && ratingRounded < 3) {
+        color = isOverall ? red : green;
```

Comment:
I think we should add `isOverall` to the dependency array because of the the build warning (probably won't affect anything functionally)